core/src/main/java/org/phoenixframework/liveview/lib/Attribute.kt

```diff
@@ -0,0 +1,50 @@
+package org.phoenixframework.liveview.lib
+
+class Attribute(private var nativeObject: Long) {
+
+    private var _name: String? = null
+
+    /** The name of an attribute */
+    val name
+        get() = _name ?: get_name(nativeObject).also { _name = it }
+    private var _namespace: String? = null
+
+    /** The namespace of an attribute */
+    val namespace
+        get() = _namespace ?: get_namespace(nativeObject).also { _namespace = it }
+
+    private var _val: String? = null
+    val value
+        get() = _val ?: get_value(nativeObject).also { _val = it }
+
+    private external fun get_name(pointer: Long): String
+
+    private external fun get_value(pointer: Long): String
+
+    private external fun get_namespace(pointer: Long): String
+
+    override fun toString(): String {
+        return "Attribute {\nName: ${name.ifEmpty { "None" }}" +
+                "\nNamespace: ${namespace.ifEmpty { "None" }}\nValue: ${value.ifEmpty { "None" }}" +
+                "\n}"
+    }
```

Comment:
Use expression form and break the string on the `\n` to make the code match the printed form to make it easier to read what would be printed. I also added 2 spaces to indent the fields when printed.

```suggestion
    override fun toString(): String =
      "Attribute {\n"+
      "  Name: ${name.ifEmpty { "None" }}\n" +
      "  Namespace: ${namespace.ifEmpty { "None" }}\n" +
      "  Value: ${value.ifEmpty { "None" }}\n" +
      "}"
```